homeassistant/components/switcher_kis/__init__.py

```diff
@@ -157,9 +163,9 @@ def mac_address(self) -> str:
         """Switcher device mac address."""
         return self.data.mac_address  # type: ignore[no-any-return]
 
-    async def async_setup(self) -> None:
+    def async_setup(self) -> None:
```

Comment:
Annotate this with `@callback` (imported from `homeassistant.core`) to indicate this function is safe to run in the event loop.